patches/net/minecraft/world/level/block/TripWireBlock.java.patch

```diff
@@ -1,11 +1,19 @@
 --- a/net/minecraft/world/level/block/TripWireBlock.java
 +++ b/net/minecraft/world/level/block/TripWireBlock.java
+@@ -3,6 +_,7 @@
+ import com.mojang.serialization.MapCodec;
+ import com.mojang.serialization.codecs.RecordCodecBuilder;
+ import com.mojang.serialization.codecs.RecordCodecBuilder.Instance;
++
```

Comment:
it seems the imports were unintentionally touched here.

Reply:
yeah that was when I did IDE rename on TollActions... big mistake. Intellij optimized imports and I had to go fix them by hand.